backend/src/components/projects/getAllProjects.js

```diff
@@ -0,0 +1,55 @@
+const axios = require('axios');
+const yamlFront = require('yaml-front-matter');
+const get = require('lodash.get');
+
+const { GH_TOKEN } = require('../../utils/consts');
+const valuesOrNull = require('../../utils/valuesOrNull');
+const formatProject = require('./formatProject');
+const reposQuery = require('../../utils/reposQuery.js');
+
+module.exports = async () => {
+  const { data: { data } } = await axios.post(
+    'https://api.github.com/graphql',
+    {
+      query: reposQuery
+    },
+    {
+      headers: {
+        Authorization: `bearer ${GH_TOKEN}`
+      },
+    }
+  );
+
+  const repositories = get(data, "organization.repositories.nodes");
+
+  if (!repositories) {
```

Comment:
be more explicit please

Reply:
Same here. I see no point for being especially explicit